src/blog/blog.controller.ts

```diff
@@ -40,4 +40,15 @@ export class BlogController {
       }
     }
   }
+
+  // Read
+  @Get('/all')
+  async getAllBoards(): Promise<Board[]> {
+    return this.boardService.getAllBoards();
+  }
+
+  @Get('/:author')
```

Comment:
P1: 이게 엔드포인트가 /blog/:author가 되는데, 실제로는 /blog/dorito 요런 식으로 엔드포인트가 나올겁니다. 특정 사용자가 가져온 블로그 목록이라는 사실을 엔드포인트로 알기 어렵습니다.
```suggestion
  @Get('/author/:author')
```
과 같은 모양이 조금 더 자연스러운 것 같습니다.

Reply:
해결했습니다! !